docs/migrating_versions/migrating_to_v050.rst

```diff
@@ -0,0 +1,89 @@
+.. _migrate_to_050:
+
+Migrating to v0.5.0 from v0.4.6
+===============================
+
+The update from v0.4.6 of Fairlearn has brought some major changes. This
+document goes through the adjustments required.
```

Comment:
We also changed a whole bunch of members from `_<name>` to `<name>_`. The CHANGES.md should have some information on that. At the very least (if you don't want to enumerate them all) we should be pointing out that any issue related to "`obj._<name>` ... does not exist on type ..." might be fixable by appending the underscore instead of preprending it.

Reply:
Added section about that. No, I'm not going to try tracking down and listing all of them :-)

Reply:
The way you wrote it it's very intuitive to me. Hopefully that'll help folks :-) I'll close this.